### ISSUE TITLE
Issue: UI is missing option to change Flash Attention KV quantization setting

### ISSUE THREAD

**Issue by GabeAl** [lmstudio-ai/lmstudio-bug-tracker#104]
I accidentally posted a bug in the cli version of the bug tracker, 

Bug fix: Flash Attention - KV cache quantization is stuck at FP16 with no way to revert to Q4_0 

This leads to a large series of usability regressions. 
1. Impossible to load 128k context on GPU
2. Slower performance even with the reduced context that fits on the GPU
3. Inability to analyze long documents with the same accuracy as before (which is ironic considering 0.3's aim to enable this)
4. All issues associated with rolling back to 0.2X series, including chat history import. 

Feel free to close the other one I made earlier: 
https://github.com/lmstudio-ai/lms/issues/70

**Comment by dmatora** [comment]
I had to switch to ollama to use 70B 128K

**Comment by yagil** [comment]
@dmatora @GabeAl is the specific ask here a way to set the KV cache quantization level?

**Comment by dmatora** [comment]
yes

**Comment by GabeAl** [comment]
Yes absolutely. Many users have commented including on Discord about this. It is one major bottleneck holding users back. 

I depend on large contexts, and large contexts need quantization (q4) to be feasible on commodity hardware.
